pulsar-broker/src/main/java/com/yahoo/pulsar/broker/lookup/LookupResult.java

```diff
@@ -30,41 +30,41 @@
  */
 public class LookupResult {
     enum Type {
-        BrokerUrl, HttpRedirectUrl
+        BrokerUrl, RedirectUrl
     }
 
     private final Type type;
     private final LookupData lookupData;
-    private final URI httpRedirectAddress;
```

Comment:
yes.. as we were not returning ```httpRedirectAddress``` to client and it was being used in internal method to find out final returning result.
- If ```Type== HttpRedirectUrl``` then redirect-http request to URL ```httpRedirectAddress```.

However, in new change all urls ```service/web-service/tls``` will be part of   ```LookupData``` and redirection will be determined using ```RedirectUrl``` type.